extras/dcgoss/dcgoss

```diff
@@ -34,8 +34,22 @@ run(){
     [[ -e "${GOSS_FILES_PATH}/goss_wait.yaml" ]] && install -m ugo+rw "${GOSS_FILES_PATH}/goss_wait.yaml" "$tmp_dir"
     [[ -n "${GOSS_VARS}" ]] && [[ -e "${GOSS_FILES_PATH}/${GOSS_VARS}" ]] && install -m ugo+rw "${GOSS_FILES_PATH}/${GOSS_VARS}" "$tmp_dir"
 
-    info "Starting docker container"
-    docker-compose run -d -T --name "$service" -v "$tmp_dir:/goss" "$service"
+    # Switch between mount or cp files strategy
+    GOSS_FILES_STRATEGY=${GOSS_FILES_STRATEGY:="mount"}
+    case "$GOSS_FILES_STRATEGY" in
+      mount)
+        info "Starting docker container"
+        docker-compose run -d -T --name "$service" -v "$tmp_dir:/goss" "$service"
+        ;;
+      cp)
+        info "Creating docker container"
+        docker-compose run -d -T --name "$service" "$service"
+        info "Copy goss files into container"
+        docker cp $tmp_dir/. $service:/goss
```

Comment:
```
In dcgoss line 48:
        docker cp $tmp_dir/. $service:/goss
                  ^------^ SC2086: Double quote to prevent globbing and word splitting.
                             ^------^ SC2086: Double quote to prevent globbing and word splitting.
```